pyproject.toml

```diff
@@ -56,7 +56,7 @@ full = [
 fuse = ["fusepy"]
 gcs = ["gcsfs"]
 git = ["pygit2"]
-github = ["requests"]
+github = ["fsspec[http]", "requests"]
```

Comment:
I've added the http extra here since now the github backend depends on HTTPFileSystem which requires aiohttp.

Reply:
Can a package depend on itself? Maybe list aiohttp here instead. 

But actually it's optional - for normal small files, you don't need it. Do you think there's an easy way to defer the import and use of HTTPFileSystem (+ appropriate error message if it fails)?